packages/dart/test/src/objects/parse_object_test.dart

```diff
@@ -4,7 +4,8 @@ import 'package:mockito/annotations.dart';
 import 'package:mockito/mockito.dart';
 import 'package:parse_server_sdk/parse_server_sdk.dart';
 import 'package:test/test.dart';
-import 'parse_query_test.mocks.dart';
+
+import '../../parse_query_test.mocks.dart';
```

Comment:
Why is this change needed if the PR doesn't add a new file?

Reply:
Because I moved the test file to a new location
from `test/parse_object_test.dart` to `test/src/objects/parse_object_test.dart`

Its recommended to have your tests folder structure follow the lib folder structure.

In this case, I moved the` parse_object_test.dart` file to a location that mirrors its location in the lib folder.

Reply:
> Because I moved the test file to a new location from `test/parse_object_test.dart` to `test/src/objects/parse_object_test.dart`
> 
> Its recommended to have your tests folder structure follow the lib folder structure.
> 
> In this case, I moved the` parse_object_test.dart` file to a location that mirrors its location in the lib folder.

@mtrezza 
I think it's good because the test section is getting bigger